content/implementations/jvm.md

```diff
@@ -19,26 +19,26 @@ Please see the [General Reference](/gherkin/gherkin-reference/) for functionalit
 Cucumber-JVM supports Java, Kotlin and Android.
 
 Other JVM languages have been moved to their own repository:
-* [Clojure](https://github.com/cucumber/cucumber-jvm-clojure)
-* [Gosu](https://github.com/cucumber/cucumber-jvm-gosu)
-* [Groovy](https://github.com/cucumber/cucumber-jvm-groovy)
-* [JRuby](https://github.com/cucumber/cucumber-jvm-jruby)
-* [Jython](https://github.com/cucumber/cucumber-jvm-jython)
-* [Rhino](https://github.com/cucumber/cucumber-jvm-rhino)
-* [Scala](https://github.com/cucumber/cucumber-jvm-scala)
+- [Clojure](https://github.com/cucumber/cucumber-jvm-clojure)
+- [Gosu](https://github.com/cucumber/cucumber-jvm-gosu)
+- [Groovy](https://github.com/cucumber/cucumber-jvm-groovy)
+- [JRuby](https://github.com/cucumber/cucumber-jvm-jruby)
+- [Jython](https://github.com/cucumber/cucumber-jvm-jython)
+- [Rhino](https://github.com/cucumber/cucumber-jvm-rhino)
+- [Scala](https://github.com/cucumber/cucumber-jvm-scala)
 
 
 ## Installation
 
-Cucumber-JVM consists of several modules (JARs) that you can download from the [public Maven repo](http://repo1.maven.org/maven2/info/cukes/).
+Cucumber-JVM consists of several modules (JARs) that you can download from the [public Maven repo](http://repo1.maven.org/maven2/io/cucumber/).
```

Comment:
Perhaps out of scope for this PR but I think we should recommend maven/gradle or any other dependency management tool here. Don't want people downloading jars manually.

Reply:
Definitely need Maven introduced here. Explanations further on rely on it.

Reply:
Working on it ;)